Content.Shared/_ES/Masks/Traitor/Components/ESAddMaskOnUseComponent.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Content.Shared._ES.Masks;
+using Content.Shared._ES.Masks.Components;
+using Content.Shared.EntityEffects;
+using Robust.Shared.Player;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server._ES.Masks.Traitor.Components;
+
+/// <summary>
+/// Adds a mask upon use of an entity
+///
+/// </summary>
+[RegisterComponent]
```

Comment:
component needs the NetworkedComponent attribute so it appears on the client correctly without delay.